components/ui-api-layer/internal/gqlschema/serviceinstance.go

```diff
@@ -3,14 +3,14 @@ package gqlschema
 import "time"
 
 type ServiceInstance struct {
-	Name                    string
-	Environment             string
-	ServiceClassName        *string
-	ServiceClassDisplayName string
-	ServicePlanName         *string
-	ServicePlanDisplayName  string
-	ServicePlanSpec         *JSON
-	CreationTimestamp       time.Time
-	Labels                  []string
-	Status                  ServiceInstanceStatus
+	Name                    string                 `json:"name"`
+	Environment             string                 `json:"environment"`
+	ServiceClassName        *string                `json:"serviceClassName"`
+	ServiceClassDisplayName string                 `json:"ServiceClassDisplayName"`
+	ServicePlanName         *string                `json:"servicePlanName"`
+	ServicePlanDisplayName  string                 `json:"servicePlanDisplayName"`
+	ServicePlanSpec         *JSON                  `json:"servicePlanSpec"`
+	CreationTimestamp       time.Time              `json:"creationTimestamp"`
+	Labels                  []string               `json:"labels"`
+	Status                  *ServiceInstanceStatus `json:"status"`
```

Comment:
I think `ServiceInstanceStatus` shouldn't be nullable. Maybe it should be fixed in another issue.